### DISCUSSION TITLE
Airflow UI plots that should've been bar charts

### DISCUSSION THREAD

**Discussion by ketozhang** [apache/airflow#36083] [Ideas]
The weakest point of Airflow UI is probably it's built-in viz especially the Task Duration, Landing Times, and Task Tries plots.

Although line charts make sense for timeseries, a stacked or side-by-side bar chart would provide a better viz for identifying outliers, dag-level trends, and avoiding overlaps at the cost harder to identify task-level trends.

**Comment by potiuk** [comment]
Yes. Agree. I think those cherts are bad. PRs to improve those are most welcome. Airflow has > 2700 contributors and anyone - especially if they care about it - can add PRs and make it happen. Those kind of improvements would be absolutely welcome.

I think maybe companies and people who would like to improve them could donate some time of their UI developers to improve that - that would be fabulous.
